strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/domain/CronTaskConfigurationDto.java

```diff
@@ -23,7 +24,7 @@
 
     private String name;
 
-    private String jobClass;
+    private Class<? extends Job> jobClass;
```

Comment:
@fuss86 are you sure this will be safety? I mean that this class is `Serializable` and it will be not good to serialize all the `Class` object within `CronTaskConfigurationDto`.

Reply:
`jackson` handles fine this kind of serialization/deserialization

> @fuss86 are you sure this will be safety?

What are your doubts ?